packages/financial-templates-lib/src/price-feed/MedianizerPriceFeed.js

```diff
@@ -6,15 +6,18 @@ class MedianizerPriceFeed extends PriceFeedInterface {
    * @notice Constructs new MedianizerPriceFeed.
    * @param {List} priceFeeds a list of priceFeeds to medianize. All elements must be of type PriceFeedInterface.
    *      Must be an array of at least one element.
+   * @param {Boolean=false} computeMean Set this to true to return the mean over price feeds instead of the median.
+   *      Default behavior is to return median.
    */
-  constructor(priceFeeds) {
+  constructor(priceFeeds, computeMean = false) {
```

Comment:
I think this is fine for now as it limits code churn. As discussed IRL we should consider refactoring this file to `CombinerPriceFeed` or `PriceFeedAggregator` as this is no really a **Median**izer any more :)

Reply:
Totally agreed